### ISSUE TITLE
Proto compiler does not work (MSVC Build)

### ISSUE THREAD

**Issue by arth7s3** [semlanik/qtprotobuf#266]
Proto compiler built with MSVC does not work, returns "system cannot find file." when parsing proto file.
The proto compiler does parse the proto file though since it returns errors found in proto file. 

Works fine with GCC.

**Comment by gmabey** [comment]
@arth7s3 on windows, I have found it useful to make sure that `protoc.exe` is on the PATH.

**Comment by Rightbputnow1** [comment]
@gmabey i have the same issue, protoc.exe is on the PATH variable. It is actually parsing the proto files (try n Introduce any syntax error in the proto file for example) but it is still returning the afore mentioned error.

**Comment by gmabey** [comment]
@Rightbputnow1 yes you're right -- I probably should have deleted my comment when I realized it didn't solve the problem, but at least I can confirm the behavior :-/
